src/Compilers/CSharp/Portable/Symbols/SpecialTypeExtensions.cs

```diff
@@ -63,6 +63,29 @@ public static bool CanBeConst(this SpecialType specialType)
             }
         }
 
+        public static bool IsValueType(this SpecialType specialType)
```

Comment:
There is an existing `IsBlittable` extension method, thanks for pointing that out. `IsBlittable` does not include `SpecialType.System_Decimal` however. But I will move this extension method to that same `SpecialTypeExtensions` helper class in the shared assembly.